plugins/woocommerce-blocks/tests/e2e/tests/checkout/checkout-block.shopper.block_theme.side_effects.spec.ts

```diff
@@ -474,3 +474,37 @@ test.describe( 'Checkout Form Errors', () => {
 		).toBeVisible();
 	} );
 } );
+
+test.describe( 'Billing Address Form', () => {
+	test.use( { storageState: guestFile } );
+	const shippingTestData = {
+		firstname: 'Jane',
+		lastname: 'Doe',
+		company: 'WooCommerce',
+		addressfirstline: '123 Main Avenue',
+		addresssecondline: 'Unit 42',
+		city: 'Los Angeles',
+		phone: '987654321',
+		country: 'Albania',
+		state: 'Berat',
+		postcode: '1234',
+	};
+	test( 'Guest user will get empty billing address form', async ( {
+		frontendUtils,
+		page,
+		checkoutPageObject,
+	} ) => {
+		await frontendUtils.goToShop();
+		await frontendUtils.addToCart( SIMPLE_PHYSICAL_PRODUCT_NAME );
+		await frontendUtils.goToCheckout();
+		await checkoutPageObject.fillShippingDetails( shippingTestData );
+		await page.getByLabel( 'Use same address for billing' ).uncheck();
+		await expect( page.locator( '#billing-first_name' ) ).toHaveValue( '' );
```

Comment:
We set 10 values but are only checking for 7. Maybe set a mapping object with selectors, and iterate on that would make this more dynamic and future proof ;)

Reply:
Fixed it in https://github.com/woocommerce/woocommerce/pull/44347/commits/a2a13150ea43dfe899c1be438ea03d026d7bcd8e